content/en/events/2023/kcseu/faq.md

```diff
@@ -0,0 +1,122 @@
+- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I have an idea for a session](#i-have-an-idea-for-a-session)
+- [My SIG wants to meet at the Summit](#my-sig-wants-to-meet-at-the-summit)
+- [Will there be a doc sprint?](#will-there-be-a-doc-sprint)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [Can I bring a guest to the Social?](#can-i-bring-a-guest-to-the-social)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+May 18th, the Tuesday of [KubeCon Europe].
+
+[KubeCon Europe]: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.rai.nl/en" rel="noopener noreferrer" target="_blank">RAI Amsterdam</a>,
+See the [Location & Venue] section for more information.
+
+[Location & Venue]: /events/2023/kcseu/location/
+
+### Will I be able to attend remotely?
+
+No, based on our experience with Kubecon NA 2021, we do not expect to be able
+to support remote access for this event.
+
+### Why is vaccination required to attend in-person?
+
+As a co-located event, the summit is required to adhere to KubeCon’s
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/attend/health-and-safety/" rel="noopener noreferrer" target="_blank">
+Health & Safety Guidelines</a>. There may be additional restrictions that
+develop closer to the event, and we will update the event accordingly to follow
+venue, municipality and CDC guidelines at a minimum.
+
+### Do I need to wear a mask?
+
+Masks are recommended, but not required for event attendees.
+
+### Why do I need to be a Kubernetes Org member to attend in-person?
+
+With uncertain attendance and staffing, we made this decision to ensure we can
+put on an event to the best of our ability.
+
+Kubernetes Orgs:
+
+<ul>
+<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
+<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
+<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
+<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
+</ul>
+
+If you are not yet a member, but active within the project, please consider
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information. If your application to join is pending, please
+contact us at community@kubernetes.io about registering.
+
+[org membership]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
```

Comment:
```suggestion
We welcome everyone to start contributing to the project, but with 1600+
Kubernetes Org members and limited space and budget requirements we've
made the decision to limit attendance to those that are already actively
engaged with the project.

If you are not yet a member, but active within the project, please consider
joining! The process of joining is straight forward. See our [org membership]
docs for more information. If your application to join is pending, please
contact us at community@kubernetes.io about registering.

Kubernetes Orgs:

<ul>
<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
</ul>


[org membership]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
```


We updated the language for NA last year, and it should probably be the thing to carry forward 👍